docs/LANGUAGE_TRANSLATION.md

```diff
@@ -0,0 +1,188 @@
+
+Here's the basic format of the spreadsheet:
+
+| id | eng | spa | 
+| :--------- | :-------- |:--------- |
+| lang_name | English | Español |
+| lang_name_in_eng | English | Spanish |
+| language | Language | Idioma |
+| welcome | Welcome! | ¡Bienvenidos! |
+
+The first column (e.g. `lang_name`) and first row (e.g. `eng`) are **required** and must be filled out for any added row/column. The rest of the cells can be filled in as new translated terms are available.
+
+### Glossary
+
+ * **Languages:** Pretty self-explanatory (a column in the spreadsheet)
+ * **Terms:** A word or set of words that need to be translated (a row in the spreadsheet). The term is identified by the `id` column
+ * **Translated Terms:** An individual translated term (a cell in the spreadsheet)
+
+### Special links
+
+If you want to link to a specific language (say you're sending the link to somebody you know speaks Somali) you can add `?lang=XXX` to the end of the URL. Example: 
+
+https://twin-cities-mutual-aid.org/?lang=som
+
+If you want to preview languages that have been created but not enabled, add `?dev` to the end of the url:
+
+https://twin-cities-mutual-aid.org/?dev
+
+---
+
+## Maintaining Translations
+
+### Translating existing terms
+
+This is fairly straightforward, just add the term to the correct place in the spreadsheet :). Checklist:
+
+ - [ ] The translated term is added to the correct column/row of the spreadsheet
+
+### Creating a new term
+
+Checklist:
+
+ - [ ] Create a new row in the spreadsheet for the new term
+ - [ ] Make sure that there is a **unique** name in the `id` column, `formatted_like_this`
+ - [ ] Make sure the term is also added to the correct place in the code, using either the `data-translation-id` attribute (in markup) or the `get` method (js code). See technical details below for more on this.
+
+### Adding a new language
+
+Adding a language involves a few steps:
+
+| Action | Edit Spreadsheet | Update code |
+| :-----------------| :-: | :-: |
+| 1. Adding a column to the spreadsheet | Yes | No |
+| 2. Adding a new flag image | No | Yes |
+| 3. Enabling the language | No | Yes | 
+
+#### Adding a column to the spreadsheet
+
+Checklist:
+
+ - [ ] The language header should contain a 3-letter code like `som` (a list of these can be [found here](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes)) 
+
+#### Adding a new flag image
+
+A good resource for hard-to-find flag images:
+
+https://en.wikipedia.org/wiki/Unrepresented_Nations_and_Peoples_Organization#Members
+
+A checklist for adding new flag images
+
+ - [ ] The image should be a PNG
+ - [ ] The image should be 23px wide
+ - [ ] The image name should have the format `lang-XXX.png` where "XXX" is the 3-letter language code
+ - [ ] The image should be placed in the `/public/images` directory in this repo
```

Comment:
Per the conversation in slack. What to do if there isn't a flag... Maybe fallback to an SVG of the 3 letter 639-2 iso code?

Reply:
I like the idea of a 3 letter fallback.